airbyte-integrations/connectors/destination-mssql-v2/src/test/kotlin/io/airbyte/cdk/load/data/sql/AirbyteTypeToSqlTypeTest.kt

```diff
@@ -0,0 +1,180 @@
+/*
+ * Copyright (c) 2024 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.data.sql
+
+import com.fasterxml.jackson.databind.JsonNode
+import io.airbyte.cdk.load.data.ArrayType
+import io.airbyte.cdk.load.data.ArrayTypeWithoutSchema
+import io.airbyte.cdk.load.data.BooleanType
+import io.airbyte.cdk.load.data.DateType
+import io.airbyte.cdk.load.data.FieldType
+import io.airbyte.cdk.load.data.IntegerType
+import io.airbyte.cdk.load.data.NumberType
+import io.airbyte.cdk.load.data.ObjectType
+import io.airbyte.cdk.load.data.ObjectTypeWithEmptySchema
+import io.airbyte.cdk.load.data.ObjectTypeWithoutSchema
+import io.airbyte.cdk.load.data.StringType
+import io.airbyte.cdk.load.data.TimeTypeWithTimezone
+import io.airbyte.cdk.load.data.TimeTypeWithoutTimezone
+import io.airbyte.cdk.load.data.TimestampTypeWithTimezone
+import io.airbyte.cdk.load.data.TimestampTypeWithoutTimezone
+import io.airbyte.cdk.load.data.UnionType
+import io.airbyte.cdk.load.data.UnknownType
+import io.mockk.mockk
+import java.sql.Types
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Assertions.assertNotNull
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.assertThrows
+
+class AirbyteTypeToSqlTypeTest {
+
+    private lateinit var converter: AirbyteTypeToSqlType
+
+    @BeforeEach
```

Comment:
nit: this thing isn't stateful, why not just `private val converter = AIrbyteTypeToSqlType()`?

Reply:
Could do that...I typically follow the setup pattern just to ensure that we are using an object specific for the test in case state does creep in at some point.  Happy to change.

Reply:
Done.